### PR TITLE
add support for docker events

### PR DESCRIPTION
**Pull request by larohra**
Execute and capture both `docker events` and `docker images` to get a better result for the images that we capture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,13 @@ https://learn.microsoft.com/en-us/azure/devops/extend/get-started/node?view=azur
 
 ## Testing changes on local environment
```
Comment (anaismiller) [inline README.md]: This says 'Testing changes on local environment'. Should it be 'Release new version'?

File: README.md [hunk 2]
```diff
@@ -8,12 +8,13 @@ https://learn.microsoft.com/en-us/azure/devops/extend/get-started/node?view=azur
...
-2. From PowerShell/bash shell, run from the etension root directory 
+1. Once the changes are commited, Update the version by modifying the VERSION file.
+2. Execute the `ReplaceVersion.ps1` script to update the version in `vss-extension.json` file and the `DfDExtension/DfDPostJobExtension.yml` file.
+3. From PowerShell/bash shell, run from the etension root directory 
     > npx tfx-cli extension create
-3) Publish and share the extension from marketplace
+4. Publish and share the extension from marketplace
 `https://marketplace.visualstudio.com/manage/publishers/<Your-publisher-name>`
-4) from ADO organization settings/Extension - add the extension. More information on publishing extension can be found [here](https://learn.microsoft.com/en-us/azure/devops/extend/publish/overview?view=azure-devops).
+5. from ADO organization settings/Extension - add the extension. More information on publishing extension can be found [here](https://learn.microsoft.com/en-us/azure/devops/extend/publish/overview?view=azure-devops).
```
Comment (anaismiller) [inline README.md]:
```suggestion
5. From ADO organization settings/Extension - add the extension. More information on publishing extension can be found [here](https://learn.microsoft.com/en-us/azure/devops/extend/publish/overview?view=azure-devops).
```

File: README.md [hunk 3]
```diff
@@ -8,12 +8,13 @@ https://learn.microsoft.com/en-us/azure/devops/extend/get-started/node?view=azur
 
 ## Testing changes on local environment 
 
-1. Once the changes are commited, Update the file vss-etension.json and the version
-2. From PowerShell/bash shell, run from the etension root directory 
+1. Once the changes are commited, Update the version by modifying the VERSION file.
+2. Execute the `ReplaceVersion.ps1` script to update the version in `vss-extension.json` file and the `DfDExtension/DfDPostJobExtension.yml` file.
+3. From PowerShell/bash shell, run from the etension root directory
```
Comment (anaismiller) [inline README.md]:
```suggestion
3. From PowerShell/bash shell, run from the extension root directory 
```

File: README.md (hunk 1 above)
Comment (anaismiller) [reply README.md]: However, not fully sure as it says '<Your-publisher-name>'

File: ReplaceVersion.ps1 [hunk 4]
```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = "Stop"
```
Comment (anaismiller) [inline ReplaceVersion.ps1:2]:
```suggestion
# Updates the version in `vss-extension.json` file and the `DfDExtension/DfDPostJobExtension.yml` file.
$ErrorActionPreference = "Stop"
```

Comment (larohra) [reply ReplaceVersion.ps1:2]: Thanks :)

File: README.md (hunk 1 above)
Comment (larohra) [reply README.md]: Modified

**Review by anaismiller — APPROVED** [review]

**Pull request merged**